src/main/java/cromwell/binding/DiskType.java

```diff
@@ -0,0 +1,15 @@
+package cromwell.binding;
+
+public enum DiskType {
```

Comment:
Built in `DiskType.values`. And never been a fan of the limited functionality in `scala.Enumeration`.